lldb/include/lldb/ValueObject/DILLexer.h

```diff
@@ -28,12 +28,14 @@ class Token {
     arrow,
     coloncolon,
     eof,
+    floating_constant,
```

Comment:
Nit: I prefer 'float_constant' to 'floating_constant'.  But I don't feel strongly about this.